internal/dsync/drwmutex.go

```diff
@@ -151,17 +151,15 @@ type Options struct {
 // blocks until either the mutex becomes available and return success or
 // more time has passed than the timeout value and return false.
 func (dm *DRWMutex) GetLock(ctx context.Context, cancel context.CancelFunc, id, source string, opts Options) (locked bool) {
-	isReadLock := false
-	return dm.lockBlocking(ctx, cancel, id, source, isReadLock, opts)
+	return dm.lockBlocking(ctx, cancel, id, source, false, opts)
 }
 
 // RLock holds a read lock on dm.
 //
 // If one or more read locks are already in use, it will grant another lock.
 // Otherwise the calling go routine blocks until the mutex is available.
 func (dm *DRWMutex) RLock(id, source string) {
-	isReadLock := true
-	dm.lockBlocking(context.Background(), nil, id, source, isReadLock, Options{
+	dm.lockBlocking(context.Background(), nil, id, source, true, Options{
```

Comment:
No reason to remove this code, please revert - it is written this way for readability.